docker/wiremock/mappings/index.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Is that needed?

Reply:
No happy to remove, I wanted to show different way of stubbing it's not required.